src/app/design/components/popover-menu/popover-menu.component.scss

```diff
@@ -0,0 +1,95 @@
+.popover-menu--container {
```

Comment:
Have you check name selector in cscc and TS files ? In scss selector name is fcs-popover.... And html app-popver